spec/sidekiq/pool_spec.rb

```diff
@@ -1,5 +1,21 @@
 describe Sidekiq::Pool do
-  it 'has a version number' do
-    expect(Sidekiq::Pool::VERSION).not_to be nil
+  describe '.after_fork_hooks' do
+    subject { described_class.after_fork_hooks }
+
+    it { is_expected.to be_instance_of(Array) }
+    it { is_expected.to be_empty }
+
+    context 'when hooks are configured' do
+      before do
+        described_class.after_fork {}
```

Comment:
If you change this to:

```ruby
Thread.new { described_class.after_fork {} }
Thread.new { described_class.after_fork {} }
```

This may create a race condition that would result in only one hook present in the array, whichever one was added the last.